JS/Lesson34/goksel/recipe-app/src/modules/recipes/NewRecipe/NewRecipeForm/index.tsx

```diff
@@ -0,0 +1,249 @@
+import { useForm } from "react-hook-form";
+import { MealCategory } from "./newRecipeHelper";
+import { MealArea } from "./newRecipeHelper";
+import { handlestrIngredientsAndMeasures } from "./newRecipeHelper"
+import { Button, FormControl, FormLabel, InputLabel, Grid, MenuItem, Select, Typography, RadioGroup, FormControlLabel, Radio } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import { Areas, Categories } from "../../models";
+
+
+export const NewRecipe = () => {
+
+    interface DataProps {
```

Comment:
Move the interface outside NewRecipe component, it will make the code more readable

Reply:
I moved it to the newRecipeHelper file.